pyquil/operator_estimation.py

```diff
@@ -913,56 +927,50 @@ def measure_observables(qc: QuantumComputer, tomo_experiment: TomographyExperime
                       DeprecationWarning)
         symmetrize_readout = readout_symmetrize
 
+    if symmetrize_readout is None:
+        symmetrize_readout = SymmetrizationLevel.NONE
+        warnings.warn("'symmetrize_readout' should now be an int, 0 instead of none.",
+                      DeprecationWarning)
+    elif symmetrize_readout == 'exhaustive':
```

Comment:
what happens if someone passes a string that is not "exhaustive"?

Reply:
I think it matches previous behavior, you just get an error that tells you which values are supported.